src/components/Article.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Article({ title, date, text }) {
+  return (
+    <article>
+      <h1>{title}</h1>
+      <span>{date}</span>
+      <p>{text}</p>
+    </article>
+  );
+}
+
+Article.propTypes = {
+  title: PropTypes.string,
+  date: PropTypes.string,
+  text: PropTypes.string,
+};
+
+Article.defaultProps = {
```

Comment:
As all the props are required , you do not need `Article.defaultProps`
and also update `Article.propTypes` as required.